docs/reference/avalanchego/info-api.md

```diff
@@ -178,8 +178,15 @@ info.getNodeID() -> {
 }
 ```
 
-- `nodeID` is this node's ID
-- `nodePOP` is this node's BLS key and proof of possession
+- `nodeID` Node ID is the unique identifier of the node that you set to act as a validator on the 
+Primary Network. If you do not have a node, you can [delegate](https://core.app/stake/delegate/), 
+or [setup a node](https://docs.avax.network/nodes).
```

Comment:
this is reference documentation. explanatory documentation should not be include in the parameter definition. 
```suggestion
Primary Network.
```

Reply:
cc: @dubrado